src/components/ui/InternalShadowRectButton/InternalShadowRectButton.tsx

```diff
@@ -80,16 +81,23 @@ const StyledInternalButton = styled(InternalButton)<
   `}
 `;
 
-const StyledButtonWrapper = styled(OakBox)`
+const StyledButtonWrapper = styled(OakBox)<
+  OakBoxProps & {
+    $hoverShadow?: OakDropShadowToken | null;
+  }
+>`
   .grey-shadow:has(+ * + .internal-button:focus-visible) {
     box-shadow: ${parseDropShadow("drop-shadow-centered-grey")};
   }
   .yellow-shadow:has(+ .internal-button:focus-visible) {
     box-shadow: ${parseDropShadow("drop-shadow-centered-lemon")};
   }
-  .yellow-shadow:has(+ .internal-button:hover) {
-    box-shadow: ${parseDropShadow("drop-shadow-lemon")};
-  }
+  ${(props) => css`
```

Comment:
Minor but this is messing with the focus styles, probably a matter of specificity

https://github.com/oaknational/oak-components/assets/122096/16793583-3318-45ec-8d85-dd6fa5bf8347

Reply:
updated